docs/views/cookies.html

```diff
@@ -26,25 +26,24 @@ <h1 class="govuk-heading-xl">Cookies</h1>
       <p>The GOV.UK Prototype Kit puts small files (known as ‘cookies’) on to your computer.</p>
 
       <p>We use cookies to:</p>
-
-      <ul class="govuk-list govuk-list--bullet">
+			<ul class="govuk-list govuk-list--bullet">
         <li>record the notifications you&#39;ve seen so we don&#39;t show them again</li>
-        <li>measure how you use the website so it can be updated and improved based on your activity</li>
+        <li>make prototypes work as well as possible</li>
       </ul>
 
       <p>These cookies aren’t used to identify you personally.</p>
 
       <p>You’ll see a message on the site before we store a cookie on your computer.</p>
 
-      <p><a href="https://ico.org.uk/for-the-public/online/cookies/">Find out how to manage cookies.</a></p>
+      <p><a href="https://ico.org.uk/for-the-public/online/cookies/">Find out how to manage cookies</a>.</p>
 
       <h2 class="govuk-heading-l">Our cookie message</h2>
 
-      <p>You will see a message about cookies when you first visit the GOV.UK Prototype Kit. We’ll store a cookie so that your computer knows you’ve seen it and knows not to show it again.</p>
+      <p>You'll see a message about cookies when you first visit the GOV.UK Prototype Kit. We’ll store a cookie so that your computer knows you’ve seen it and knows not to show it again.</p>
 
 			{{ govukTable({
 				classes: "app-table--fixed",
-			  caption: "Dates and amounts",
+			  caption: "Our cookie message",
```

Comment:
I don't think this is quite right (but it's definitely better than it was…!) – This table is listing the cookies that we set, so maybe the caption should be something like 'Cookies relating to our cookies message that we store on your computer', although that's quite verbose?

Reply:
Thanks Olli - fixed in https://github.com/alphagov/govuk-prototype-kit/pull/845/commits/63b740469df7ad1942edd594dfbda070ae8f2825